examples/example_parameters.param

```diff
@@ -0,0 +1,106 @@
+# Enhance contrast of the images mainly to cut off outliers.
+enhanceContrastSaturationPercentage 0.15
+# Minimum intensity in the image is mapped to 0
+normalizeMinToZero 1
+
+###################
+### Calculation ###
+###################
+# Rigid registration
+# ------------------
+# Pre-align images with rigid registration (translation) using cross-correlation
+useCorrelationToInitTranslation 0
+# The maximum translation in pixels of each image in this alignment. 0 = no limit on the translation
+maxCorrShift 0
+
+# Gradient descent settings
+# -------------------------
+# Maximum number of gradient descent steps
+maxGDIterations 500
+# Desired "precision" to be achieved during gradient descent
+stopEpsilon 1e-6
+# Regularization factor in the first stage, highest level
+lambda 200
+# Regularization factor update per level. For example, if you have images of level 8, then lambda will be applied in 8, lambda * lambdaFactor in 7, lambda * lambdaFactor^2 in 6...
+lambdaFactor 1
+
+# Levels
+# ------
+# Levels specify the resolution of the input images and the parameters of the multilevel minimization. The multilevel algorithm expects the input images to be quadratic and to have a dyadic resolution, i.e. the number of pixels in one coordinate direction is a power of two. The exponent is called level. For instance, level 8 means that the input images have $2^8=256$ pixels in each direction. Thus, 9 and 10 encode 512 and 1024 pixels respectively.
+# Most of these should be filled out by default if you use the python wrapper.
+# The precision of the input images = log_2(image sidelength) and the size of the output
+precisionLevel 8
+# The stopping level of the grid. i.e. deformations are not calculated beyond this resolution even if images have more pixels. If you want the resolution to equal the resolution of the images, stoplevel = precisionLevel. Higher levels take longer to compute.
+stopLevel 8
+# the starting level of the grid, i.e. the optimization will first be done on a grid of size 2^startLevel x 2^startlevel. startLevel should be small enough so that you still see the individual atoms if the input images are downsampled to this size. Usually startLevel is one or two smaller than stopLevel.
+startLevel 6
+# In the refinement step start at this level, usually >startLevel and <=stopLevel.
+refineStartLevel 7
+# In the refinement step stop at this level
+refineStopLevel 8
+# ?
+checkboxWidth 8
+# Resize the input images to the stop level, if different from precision level.
+resizeInput 0
+
+# In terms of Algorithm 1 of the Ultramicroscopy paper, we have $m_0=$ `startLevel` and $m_1=$ `stopLevel` in the first for loop of Algorithm 2, and $m_0=$ `refineStartLevel` and $m_1=$ `refineStopLevel` for the second for loop in Algorithm 2.
+
+# Miscellaneous
+# -------------
+# Number of times the entire calculation is performed
+numExtraStages 2
+# Adjustment of lambda in subsequent stages. Usually <1 because needed adjustments are minor
```

Comment:
The reasoning behind "<1" is slightly different. In the first stage, we align all frames to the first frame of our series, i.e the reference frame is the first frame and thus as noisy as all the other frames. Starting from stage 2, when align all frames to the average of the aligned images from the preceding stage, i.e. the reference frame is the average and thus much smoother than our reference in the first stage. Hence, we need less regularization starting from stage 2, i.e. we multiply lambda by a factor <1 to reduce the regularization.